frostsnapp/native/src/api.rs

```diff
@@ -1038,6 +1038,11 @@ impl Coordinator {
     pub fn get_connected_device(&self, id: DeviceId) -> SyncReturn<Option<ConnectedDevice>> {
         SyncReturn(self.0.get_connected_device(id))
     }
+
+    pub fn wipe_device_data(&self, device_id: DeviceId) -> SyncReturn<()> {
```

Comment:
I think `SyncReturn<()>` is unnecessary. Is this thing fallible? It should probably return a Stream. I'm guessing this is just keeping it simple for now which is fine as who knows how this turns out after we do some redesigning.